docs/examples/example-app/src/test/java/org/lfenergy/compas/sct/examples/AppTest.java

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2022 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.examples;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class AppTest
+{
+    @Test
+    public void shouldAnswerWithTrue()
+    {
+        assertTrue( true );
```

Comment:
To be completed

Reply:
https://github.com/com-pas/compas-sct/pull/149/files#diff-7d0c53caaafa4cdd7f817ff8114e0c6901049abeafc477faf084349a51aa82eb